packages/ckb-sdk-core/src/signWitnesses.ts

```diff
@@ -34,23 +34,23 @@ const signWitnesses: SignWitnesses = (key: Key | Map<LockHash, Key>) => ({
   witnesses: StructuredWitness[]
   inputCells: CachedLock[]
 }) => {
-  if (!key) throw new ArgumentRequired('Private key')
+  if (!key) throw new ArgumentRequired('Signature provider')
   if (!transactionHash) throw new ArgumentRequired('Transaction hash')
   if (!witnesses.length) throw new Error('Witnesses is empty')
 
   if (isMap(key)) {
     const rawWitnesses = witnesses
     const restWitnesses = witnesses.slice(inputCells.length)
     const groupedScripts = groupScripts(inputCells)
+
     groupedScripts.forEach((indices, lockhash) => {
       const sk = key.get(lockhash)
-      if (!sk) {
-        throw new Error(`The private key to sign lockhash ${lockhash} is not found`)
-      }
-      const ws = [...indices.map(idx => witnesses[idx]), ...restWitnesses]
+      if (sk) {
```

Comment:
Please update the test case

Reply:
https://github.com/nervosnetwork/ckb-sdk-js/blob/develop/packages/ckb-sdk-core/__tests__/signWitnesses/fixtures.json#L183

Reply:
The test case is updated and passed now.